src/briefcase/commands/base.py

```diff
@@ -213,6 +215,39 @@ def validate_data_path(self, data_path):
 """
             )
 
+        if not os.path.exists(data_path):
+            try:
+                # The Windows Store version of Python can redirect filesystem
+                # interactions within %LOCALAPPDATA% to a sandboxed location.
+                # To bypass this, the Briefcase cache directory creation is
+                # performed via ``cmd.exe`` in a different process. Once this
+                # directory exists in the "real" %LOCALAPPDATA%, Windows will
+                # allow normal interactions without attempting to sandbox them.
+                if platform.system() == "Windows":
+                    subprocess.run(
+                        ["mkdir", data_path],
+                        shell=True,
+                        check=True,
+                        stdout=subprocess.DEVNULL,
+                        stderr=subprocess.DEVNULL,
+                    )
+                else:
+                    os.makedirs(data_path, exist_ok=True)
```

Comment:
Is this strictly needed? Presumably the data path will be created on first usage - this is more of a convenience/ensuring that all platforms are behaviourally consistent?

Reply:
That's correct; I only added this to keep behavior the same on all platforms.